.github/workflows/automatic-release.yml

```diff
@@ -11,6 +11,9 @@ on:
       GITHUB_USER_TOKEN:
         description: Authentication token with write permission needed by the release bot (falls back to GITHUB_TOKEN).
         required: false
+      NPM_TOKEN:
```

Comment:
Isn't that the same as the `NPM_REGISTRY_TOKEN` - see for example https://github.com/inpsyde/reusable-workflows/blob/main/.github/workflows/wp-scripts-lint.yml#L56-L58 which is put into `NODE_AUTH_TOKEN: ${{ secrets.NPM_REGISTRY_TOKEN }}` - see https://github.com/inpsyde/reusable-workflows/blob/main/.github/workflows/wp-scripts-lint.yml#L78 ?

The Github action "setup-node" will automatically read this token from `env.NODE_AUTH_TOKEN` and set it.

Reply:
It is aligned to use the `NPM_REGISTRY_TOKEN` secret and internally we use it to set `NPM_TOKEN` since this is what it is consumed by the underlying implementation see: https://github.com/semantic-release/semantic-release/blob/master/docs/usage/ci-configuration.md#authentication-for-plugins

Reply:
That's fine if `NPM_TOKEN` is used for semantic-release, but we always name our inputs/secrets the same.